spec/acceptance/init_spec.rb

```diff
@@ -8,8 +8,6 @@
         MANIFEST
 
       # Run it twice and test for idempotency
-      expect(apply_manifest(pp).exit_code).not_to eq(1)
```

Comment:
Are we seeing failures in the acceptance tests?

Reply:
I did some digging into this. It turns out this is a failure in `telegraf` version 1.15.1 where it does not set the permissions on `/var/log/telegraf` correctly, so the service cannot write logs and fails to start.

In the 1.15.1 RPM

```
# rpm -qvl telegraf
-rw-r--r--    1 root    root                      131 Jul 22 22:21 /etc/logrotate.d/telegraf
-rw-r--r--    1 root    root                   250761 Jul 22 22:21 /etc/telegraf/telegraf.conf
drwxr-xr-x    2 root    root                        0 Jul 22 22:21 /etc/telegraf/telegraf.d
-rwxr-xr-x    1 root    root                 69730912 Jul 22 22:21 /usr/bin/telegraf
drwxr-xr-x    2 root    root                        0 Jul 22 22:21 /usr/lib/.build-id
drwxr-xr-x    2 root    root                        0 Jul 22 22:21 /usr/lib/.build-id/3c
lrwxrwxrwx    1 root    root                       28 Jul 22 22:21 /usr/lib/.build-id/3c/1b944565dc487f5646d216f361977b5c6bb4c0 -> ../../../../usr/bin/telegraf
-rwxr-xr-x    1 root    root                     5803 Jul 22 22:21 /usr/lib/telegraf/scripts/init.sh
-rw-r--r--    1 root    root                      492 Jul 22 22:21 /usr/lib/telegraf/scripts/telegraf.service
drwxr-xr-x    2 root    root                        0 Jul 22 22:21 /var/log/telegraf
```

In the 1.14.5 RPM

```
# rpm -qvl telegraf
-rw-r--r--    1 root    root                      131 Jun 30 19:20 /etc/logrotate.d/telegraf
drwxr-xr-x    2 root    root                        0 Jun 30 19:21 /etc/telegraf
-rw-r--r--    1 root    root                   235766 Jun 30 19:20 /etc/telegraf/telegraf.conf
drwxr-xr-x    2 root    root                        0 Jun 30 19:21 /etc/telegraf/telegraf.d
-rwxr-xr-x    1 root    root                 69213184 Jun 30 19:20 /usr/bin/telegraf
drwxr-xr-x    2 root    root                        0 Jun 30 19:21 /usr/lib/.build-id
drwxr-xr-x    2 root    root                        0 Jun 30 19:21 /usr/lib/.build-id/87
lrwxrwxrwx    1 root    root                       28 Jun 30 19:21 /usr/lib/.build-id/87/58b4a9009b5001278739cd097e59d24c18f23e -> ../../../../usr/bin/telegraf
-rw-r--r--    1 root    root                     5803 Jun 30 19:20 /usr/lib/telegraf/scripts/init.sh
-rw-r--r--    1 root    root                      492 Jun 30 19:20 /usr/lib/telegraf/scripts/telegraf.service
drwxr-xr-x    2 telegraftelegraf                    0 Jun 30 19:21 /var/log/telegraf
```

This is not an issue in the DEB packages as there is a post install script that sets the permissions correctly: https://github.com/influxdata/telegraf/blob/master/scripts/deb/post-install.sh#L52


An issue was opened for this over in the `telegraf` repo: https://github.com/influxdata/telegraf/issues/7905